### DISCUSSION TITLE
how do ii add a new device to my device tree for an orin nano with scarthgap?

### DISCUSSION THREAD

**Discussion by arrow53** [OE4T/meta-tegra#1686] [Q&A]
I need to add a device tree entry for a pcie to Ethernet Microchip LAN7430 controller. 

Can someone give me an idea where to start? I see a bunch of dts files in `orin-nano.inc`. Should I edit one of those? I can't even find the recipe that holds the main one `tegra234-p3768-0000+p3767-0005-nv.dtb`

**Comment by madisongh** [comment]
The recipe with NVIDIA's device trees is [nvidia-kernel-oot-dtb](https://github.com/OE4T/meta-tegra/blob/master/recipes-kernel/nvidia-kernel-oot/nvidia-kernel-oot-dtb_36.3.0.bb).

**Reply by arrow53** [reply]
thanks. This whole separate oot thing is new to me. I guess makes sense they are doing it this way but pretty different than the past.
